packages/lib-classifier/src/components/Classifier/components/FieldGuide/components/FieldGuide/components/FieldGuideItem/FieldGuideItem.js

```diff
@@ -28,6 +28,19 @@ const FieldGuideItemHeader = styled(Box)`
   }
 `
 
+const FieldGuideItemContent = styled(Box)`
+  > h1, h2, h3, h4, h5, h6 {
+    font-size: 14px;
```

Comment:
Should pixel sizes like this be set in the theme? It seems odd, to me, to have components that use non-standard spacings/sizes.

Reply:
The theme's defaults are set to what Becky's defined here: https://projects.invisionapp.com/d/main#/projects/prototypes/12923997/assets/77727645

The field guide's design though shows the same size for all the sub-headings in the markdown, so, I'm setting it here, since it's the only thing so far doing something like this (the tutorial is only resetting the size for h1 and h2 only). If we find ourselves needing to repeat this somewhere else, we can either use the `extend` theme option or maybe set the small size headings in the theme to do something like this?